ls1intum/Artemis
# [Exam] Hide unnecessary navbar elements in exam mode

### Checklist

### Motivation and Context
Hide unnecessary navbar elements to distract students less while participating in an exam

### Steps for Testing

1. Log in to Artemis
2. Navigate through Artemis
3. Make sure the navbar stays unchanged unless you enter an exam which has already started and is not over, then it should look like in the screenshots below.

### Screenshots
Elements that should be hidden in exam mode:
![grafik](https://user-images.githubusercontent.com/63286106/86052968-11fe7a80-ba58-11ea-93ae-dc75b455cf6f.png)
![grafik](https://user-images.githubusercontent.com/63286106/86052016-9f40cf80-ba56-11ea-9fb5-9534548843b5.png)
